nationalarchives/ds-wagtail
# DF-664:Fix show record opening date error for invalid date in Catalogue search

Ticket URL: https://national-archives.atlassian.net/browse/DF-664

## About these changes

Changes to show record opening date error for invalid date in Catalogue search

## How to check these changes

- Navigate to Catalogue Results - Records at the National Archives
- Enter invalid date range in record opening date filter, From Date after To Date ex: From 2000, To 1999

Observe 
- the error message displayed
- No results message



## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
